distributor-api-v1/general-guidelines.md

```diff
@@ -58,47 +58,3 @@ In case of any error, the returned JSON object describes the error and has the f
-
-To send the correct [CreditCardData](operations.md#creditcarddata) you need (check the [docs](operations.md#creditcarddata) to see which fields are required):
-* Card holder name
-* Card number
-* CVV
-* Expiration date
-
-This is the relation between the human readable names and the field names which Distributor API uses:
-
-| Human readable      | Distributor API       |
-| :------------------ | :-------------------- |
-| Card holder name    | `HolderName`          |
-| Card number         | `PaymentGatewayData`  |
-| Expiration date     | `Expiration`          |
-| CVV                 | `PaymentGatewayData`  |
-
-`Expiration` and `HolderName` are not sensitive data in terms of PCI DSS and plain text can be used. The `Expiration` does need to follow the format described in [CreditCardData](operations.md#creditcarddata).
-
-[PaymentGatewayData](payment-gateway-data.md) is a string representing some card data (CVV and card number) which are encoded via payment card storage provider. Those card data are sensitive in terms of PCI DSS and therefore the implementation should not handle them in plain text. 
-
-### To add payment cards support
-
-Please be aware that names of fields in PCI Proxy and Distributor API differ but they represent the same things:
-
-| Distributor API      | PCI Proxy       |
-| :-----------------   | :------------   |
-| `PublicKey`          | `merchantId`    |
-| `PaymentGatewayData` | `transactionId` |
-
-* Confirm that property supports PCI Proxy by checking the field [PaymentCardStorageType](operations.md#paymentcardstoragetype) in Distributor API response. Also read docs about [PaymentGateway](operations.md#payment-gateway) to see what other API data you could potentially use for your implemention.
-* Use [PublicKey](operations.md#payment-gateway) as `merchantId` and PCI Proxy with their [guide](https://docs.pci-proxy.com/collect-and-store-cards/capture-iframes) to handle the sensitive card data and to obtain `PaymentGatewayData`.
-* PCI Proxy documentation only highlights the sandbox endpoints. If you want to have access to the production endpoint, you have too omit the `sandbox.` part from the address. For example `https://pay.sandbox.datatrans.com/upp/payment/js/secure-fields-2.0.0.min.js` then becomes `https://pay.datatrans.com/upp/payment/js/secure-fields-2.0.0.min.js` .
-* Use `transactionId` from PCI Proxy as `PaymentGatewayData` inside [CreditCardData](operations.md#creditcarddata). 
- 
-| Distributor API Field | Used in         | Source          |
-| :-----------------    | :------------   | :-----          |
-| `PublicKey`           | PCI Proxy       | Distributor API |
-| `PaymentGatewayData`  | Distributor API | PCI Proxy       |
-
```

Comment:
while reading this I got a very confused. So I have simplified this by the steps how to obtain the `merchantId` and how to use `transactionId` in the `use-case` doc